src/solver/types.jl

```diff
@@ -427,10 +427,18 @@ the default configurations set in `default_config_dict()`.
 function AtmosConfig(config::Dict; comms_ctx = nothing)
     config = override_default_config(config)
     FT = config["FLOAT_TYPE"] == "Float64" ? Float64 : Float32
-    toml_dict = CP.create_toml_dict(
-        FT;
-        override_file = CP.merge_toml_files(config["toml"]),
-    )
+
+    # Check that config["toml"] is a filepath and the file does not exist
+    if config["toml"] isa AbstractString && !isfile(config["toml"])
+        @info "Could not find TOML file at path $(joinpath(pwd(), config["toml"]))"
+        prepend_path = joinpath(dirname(@__FILE__), "..", "..", config["toml"])
+        if isfile(prepend_path)
+            @info "Prepending `pkgdir(CA)` to relative TOML file path"
```

Comment:
Why not `Using bundled TOML file", which is a more user-friendly warning?

Reply:
IMO "bundled" is not clear. Are you ok with "Using TOML at $prepend_path"?

Reply:
> IMO "bundled" is not clear. Are you ok with "Using TOML at $prepend_path"?

For users, that would result in

`Using TOML at `/home/sbozzolo/.julia/packages/1.9/ClimaAtmos/asd23/toml/very_long_name.toml`

The relevant information here I think is that we are using the one that comes when you install ClimaAtmos, but the path is not needed. I think we shouldn't need to print out this internal piece of information.

I don't know what's the best verbiage, maybe `Using default TOML file for job $job_id`

Reply:
I see the path being useful for users looking at a run log and being able to see which parameter filepath was used. If we don't print the path, it requires a bit of detective work for a user to figure out which file was used in a simulation.